common/djangoapps/enrollment/views.py

```diff
@@ -700,3 +707,63 @@ def post(self, request):
                     actual_activation=current_enrollment['is_active'] if current_enrollment else None,
                     user_id=user.id
                 )
+
+
+@can_disable_rate_limit
+class BulkEnrollView(APIView):
+    """
+    **Use Case**
+
+        Get the user's enrollment status for a course.
+
+    **Example Request**
+
+        POST /api/enrollment/v1/bulk_enroll/ {
+            "auto_enroll": true,
+            "email_students": true,
+            "action": "enroll",
+            "courses": "course-v1:edX+Demo+123,course-v1:edX+Demo2+456",
+            "identifiers": "brandon@example.com,yamilah@example.com"
+        }
+
+        **POST Parameters**
+
+          A POST request can include the following parameters.
+
+          * auto_enroll: When set to `true`, students will be enrolled as soon
+            as they register.
+          * email_students: When set to `true`, students will be sent email
+            notifications upon enrollment.
+          * action: Can either be set to "enroll" or "unenroll". This determines the behabior
+
+    **Response Values**
+
+        If the supplied course data is valid and the enrollments were
+        successful, an HTTP 200 "OK" response is returned.
+
+        The HTTP 200 response body contains a list of response data for each
+        enrollment. (See the `instructor.views.api.students_update_enrollment`
+        docstring for the specifics of the response data available for each
+        enrollment)
+    """
+
+    authentication_classes = JwtAuthentication, OAuth2AuthenticationAllowInactiveUser
```

Comment:
Is `OAuth2AuthenticationAllowInactiveUser` required here?  I'm not sure it's a good idea to allow inactive user accounts to interact with this workflow, assuming that's what this particular permission class enables.